docs/tutorials/sessionkeys/createsessionRouter.md

```diff
@@ -0,0 +1,216 @@
+---
+sidebar_label: "Create Session Router"
+sidebar_position: 6
+---
+
+# Create Session Router
+
+:::info
+One crucial aspect to grasp about the session module is its limitation on sending batches of various user operations. Specifically, it does not support sending multiple ERC20 Transfers with different recipients within the same batch, nor does it allow combining an ERC20 transfer with an NFT Mint operation. To overcome this constraint, we rely on the Batched Session Router Module.
```

Comment:
we can ask user to skip to the last file if they not plan to use the session router module or complete the session module tutorial first and then introduce the 2 new files.